I do magic/promises-callbacks-js/src/html/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Super Agentes - Home</title>
+    <!--Bootstrap grid-->
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.2/mustache.min.js"></script>
```

Comment:
Load all scripts in the bottom of the body section.

Reply:
Done